scripts/advancedDeviceSetup.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+echo "Updating Libraries"
+
+# Update iOS Podfile to use NeuroID/Advanced 
+version=$(sed -n "s/.*'NeuroID', '\([^']*\)'.*/\1/p" neuroid-reactnative-sdk.podspec)
+
+if [ -n "$version" ]; then
+    sed -i '' 's/s.dependency "React-Core"/s.dependency "NeuroID\/AdvancedDevice", '"'$version'"' \n  s.dependency "React-Core"/' neuroid-reactnative-sdk.podspec
+else
+    sed -i '' 's/s.dependency "React-Core"/s.dependency "NeuroID\/AdvancedDevice" \n  s.dependency "React-Core"/' neuroid-reactnative-sdk.podspec
+fi
+
+# Update Android Libraries 
+sed -i '' 's/neuroid-android-sdk:react-android-sdk/neuroid-android-sdk:react-android-advanced-device-sdk/' android/build.gradle
+
+# Implement code for iOS header and actual
+# # Add to ios header
+sed -i '' '/start:/i \
+RCT_EXTERN_METHOD(start: \
+                parameters:(BOOL *)advancedDeviceSignals \
+                withResolver: (RCTPromiseResolveBlock)resolve \
+                withRejecter:(RCTPromiseRejectBlock)reject) \
+\
+' ios/NeuroidReactnativeSdk.m
+
+# Add to ios function file
+sed -i '' '/@objc(start:withRejecter:)/i \
+    @objc(start:advancedDeviceSignals:withResolver:withRejecter:) \
+    func configure(advancedDeviceSignals: Bool, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void { \
```

Comment:
should this be named
`start(advancedDeviceSignals: Bool, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) `
instead of 
`configure(advancedDeviceSignals: Bool, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock)`

I see a corresponding extern call in theNeuroidReactnativeSdk.m file to 

```
RCT_EXTERN_METHOD(start: 
                parameters:(BOOL *)advancedDeviceSignals 
                withResolver: (RCTPromiseResolveBlock)resolve 
                withRejecter:(RCTPromiseRejectBlock)reject) 
```

Reply:
Great catch, updated